content/docs/1.13/performance-tuning.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Performance Tuning Guide
+description: Tweaking your Jaeger instance to achieve a better performance
+weight: 10
+---
+
+Jaeger was built from day 1 to be able to ingest huge amounts of data in a resilient way. To better utilize resources that might cause delays, such as storage or network communications, Jaeger buffers and batches data. When more spans are generated than Jaeger is able to safely process, spans might get dropped. However, the defaults might not fit all scenarios: for instance, agents running as a sidecar might have more memory constraints than agents running as a daemon in bare metal.
+
+## Adjusting the client settings
+
+### Settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`
+
+Together, the sampler type and parameter specify how often traces should be "sampled", ie, recorded and sent to the Jaeger backend. For applications generating a large number of spans, setting the sampling type to `probabilistic` and the value to `0.001` (the default) will cause traces to be reported with a 1/1000th chance. Note that the sampling decision is made at the root span and propagated down to all child spans.
+
+For applications with low to medium traffic, setting the sampling type to `const` and value to `1` will cause all spans to be reported. Similarly, tracing can be disabled by setting the value to `0`, while context propagation will continue to work.
```

Comment:
To our internal users we always recommend using remote sampler, so that concrete sampling strategies are decided by the  backend. Even though adaptive sampling isn't live yet in oss, the backend-driven configuration is already supported (../sampling/#collector-sampling-configuration), so perhaps worth mentioning here, as otherwise fixed configurations sound like the recommended approach.

Reply:
> We recommend to set your clients to use the [`remote` sampling strategy](./sampling/#collector-sampling-configuration), so that admins can centrally set the concrete sampling strategy for each service.

Like this?